doc/DESI_SPECTRO_REDUX/SPECPROD/exposures/NIGHT/EXPID/qframe-CAMERA-EXPID.rst

```diff
@@ -0,0 +1,225 @@
+    is the 8-digit exposure ID.
+:Regex: ``qframe-[brz][0-9]-[0-9]{8}\.fits``
+:File Type: FITS, 70 MB
+
+Contents
+========
+
+====== ========== ======== ===================
+Number EXTNAME    Type     Contents
+====== ========== ======== ===================
+HDU0_  FLUX       IMAGE    Extracted electrons (photons)
+HDU1_  IVAR       IMAGE    Inverse variance of extracted electrons
+HDU2_  MASK       IMAGE    Bad value mask; 0=good
+HDU3_  SIGMA      IMAGE    LSF sigma in CCD pixel units
+HDU4_  WAVELENGTH IMAGE    Wavelength grid of the extraction
+HDU5_  FIBERMAP   BINTABLE Fibermap
+====== ========== ======== ===================
+
+FITS Header Units
+=================
+
+HDU0
+----
+
+EXTNAME = FLUX
+
+Extracted electrons[nspec, nwave]
+
+Required Header Keywords
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+======== =========================================== ===== ==================================
+KEY      Example Value                               Type  Comment
+======== =========================================== ===== ==================================
+NAXIS1   6001                                        int   Number of wavelength samples
+NAXIS2   500                                         int   Number of extracted spectra
+NIGHT    20200316                                    str   Night of observation YEARMMDD
+EXPID    20                                          int   DESI exposure ID
+TILEID   11108                                       int   DESI tile ID
+PROGRAM  DARK                                        str   program [dark, bright, ...]
+FLAVOR   science                                     str   Flavor [arc, flat, science, zero, ...]
+TELRA    150.87                                      float Telescope pointing RA [degrees]
+TELDEC   31.23                                       float Telescope pointing dec [degrees]
+AIRMASS  1.307154717878038                           float Airmass at middle of exposure
+EXPTIME  1142.541228573218                           float Exposure time [sec]
+SEEING   0.7572662830352783                          float Seeing FWHM [arcsec]
+MOONFRAC 0.4083473802955095                          float Moon illumination fraction 0-1; 1=full
+MOONALT  -79.39563600071901                          float Moon altitude [degrees]
+MOONSEP  131.2947533254612                           float Moon:tile separation angle [degrees]
+DATE-OBS 2020-03-17T02:42:47.160'                    str   Start of exposure
+PASS     1                                           int
+RA       150.87                                      float
+DEC      31.23                                       float
+EBMV     0.0189034678041935                          float
+MJD      58925.11304582807                           float
+TRANSPAR 0.7192993760108948                          float
+DOSVER   SIM                                         str
+FEEVER   SIM                                         str
+BUNIT    electron/Angstrom                           str
```

Comment:
We've been using `count/Angstrom` because `electron` is not part of the FITS standard for units.